geomesa-cassandra/geomesa-cassandra-tools/conf-filtered/cassandra-env.sh

```diff
@@ -33,3 +33,6 @@ function get_cassandra_classpath() {
     echo "$classpath"
   fi
 }
+
+get_cassandra_classpath
+
```

Comment:
Extra whitespace (just to show that I'm reading). ;)

Reply:
i think we try to always have a trailing empty line to fix sed commands or something like that?

Reply:
Eh, I think I saw one without.... not a big deal though!

Reply:
it's aspirational :)